sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -851,11 +866,39 @@ public HttpPipeline getHttpPipeline() {
             listModelOptions.getDependenciesFor(),
             listModelOptions.getIncludeModelDefinition(),
             new DigitalTwinModelsListOptions().setMaxItemCount(listModelOptions.getMaxItemCount()),
-            context);
+            context)
+            .map(
+                objectPagedResponse -> {
+                    List<ModelData> convertedList = objectPagedResponse.getValue().stream()
+                        .map(object -> ModelDataConverter.map(object))
```

Comment:
method reference here as well -> anywhere we are calling a static method can be replaced by a method reference

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/methodreferences.html